backend/src/data/migrations/20220906143050_add_course_categories_prices_table.ts

```diff
@@ -0,0 +1,71 @@
+import { Knex } from 'knex';
+
+import { IdContainer } from '~/common/types/types';
+
+enum TableName {
+  COURSE_CATEGORIES_PRICES = 'course_categories_prices',
+  COURSE_CATEGORIES = 'course_categories',
+}
+
+enum ColumnName {
+  ID = 'id',
+  CREATED_AT = 'created_at',
+  UPDATED_AT = 'updated_at',
+  CATEGORY_ID = 'category_id',
+  PRICE = 'price',
+}
+
+const DELETE_STRATEGY = 'CASCADE';
+
+const INITIAL_CATEGORY_PRICE = 5;
+const CATEGORY_PRICE_INTIAL_DELTA = 0.99;
+
+const getRandomPrice = (): number => {
+  const countOfDecimalsAfterDot = 2;
+  const isPriceValueChangePositive = Math.random() > 0.5;
+  const priceChange = isPriceValueChangePositive
+    ? CATEGORY_PRICE_INTIAL_DELTA
+    : -CATEGORY_PRICE_INTIAL_DELTA;
+  const newPrice = INITIAL_CATEGORY_PRICE + Math.random() * priceChange;
```

Comment:
I think the price should be any number between 4.01 and 5.99, not one of them

Reply:
There was nothing in the requirements about that, sooooo, let's see, what coaches are going to say about those price limits

Reply:
"4.01 - minimum, 5.99 - maximum" - from the requirements, soooooooo, I think it should be left as it is